elevate-cpanel

```diff
@@ -6457,7 +6457,8 @@ EOS
             qr/^cloudlinux-ea4-rollout(?:-[0-9]+)?$/,
             'cl-ea4',
             qr/^cl-mysql(?:-meta)?/,
-            'mysqclient', 'mysql-debuginfo'
+            'mysqclient', 'mysql-debuginfo',
+            'cl7h'
```

Comment:
Does CL leapp handle this repo?  Typically just adding a repo to the vetted list is not acceptable as we would need to add a component to handle upgrading the packages from the repo.

Reply:
Okay, this change is fine since it is mapped in CL leapp.  We'll just need you to make the change in `lib/Elevate/OS/CloudLinux7.pm` and run make to regenerate `elevate-cpanel` since `elevate-cpanel` is purely a generated file based on `lib/scripts`